src/Assetic/Filter/CoffeeScriptFilter.php

```diff
@@ -20,19 +20,17 @@
  * @link http://coffeescript.org/
  * @author Kris Wallsmith <kris.wallsmith@gmail.com>
  */
-class CoffeeScriptFilter extends BaseNodeFilter
+class CoffeeScriptFilter extends BaseProcessFilter
 {
     private $coffeeBin;
-    private $nodeBin;
 
     // coffee options
     private $bare;
     private $noHeader;
 
-    public function __construct($coffeeBin = '/usr/bin/coffee', $nodeBin = null)
+    public function __construct($coffeeBin = 'coffee')
```

Comment:
More specifically, the user can pass `'coffee'` as the first argument, and there's no reason to remove the second argument. I've defaulted to an absolute path because the web server doesn't always have the same PATH as the CLI account.